addons/mail/static/src/core/web/activity_list_popover.js

```diff
@@ -84,7 +84,7 @@ export class ActivityListPopover extends Component {
             "activity_format",
             [props.activityIds],
             {
-                context: this.user.user_context,
+                context: user.context,
```

Comment:
Isn't/shouldn't the user context be automatically provided to the orm calls instead?

Reply:
it is indeed, good catch :+1: